Tests/TuistCacheTests/ContentHashing/CoreDataModelsContentHasherTests.swift

```diff
@@ -0,0 +1,91 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistCoreTesting
+import XCTest
+import TuistSupport
+import TuistCacheTesting
+@testable import TuistCache
+@testable import TuistSupportTesting
+
+
+final class CoreDataModelsContentHasherTests: TuistUnitTestCase {
+    private var sut: CoreDataModelsContentHasher!
+    private var coreDataModel: CoreDataModel!
+    private var temporaryDirectory: TemporaryDirectory!
+    private var mockContentHasher: MockContentHashing!
+    private let defaultValuesHash = "05c9d517e2cf12b45786787dae929a23" // Expected hash for the CoreDataModel created with the buildCoreDataModel function using default values
+    override func setUp() {
+        super.setUp()
+        mockContentHasher = MockContentHashing()
+        sut = CoreDataModelsContentHasher(contentHasher: mockContentHasher)
+        do {
+            temporaryDirectory = try TemporaryDirectory(removeTreeOnDeinit: true)
+        } catch {
+            XCTFail("Error while creating temporary directory")
+        }
+    }
+
+    override func tearDown() {
+        sut = nil
+        coreDataModel = nil
+        temporaryDirectory = nil
+        mockContentHasher = nil
+        super.tearDown()
+    }
+
+    // MARK: - Tests
+
+    func test_hash_returnsSameValue() throws {
+        coreDataModel = try buildCoreDataModel()
+        mockContentHasher.hashStringsStub = "fixed"
+
+        let hash = try sut.hash(coreDataModels: [coreDataModel])
+
+        XCTAssertEqual(hash, "fixed")
+    }
+
+    func test_hash_fileContentChangesHash() throws {
+        let name = "CoreDataModel"
```

Comment:
Not a big deal but we try to organize the content of each test method in 3 blocks: given, when, then. It makes it easier to understand what's each piece of the test definition.

Reply:
Applied to all tests